traffic_ops/traffic_ops_golang/deliveryservice/eligible.go

```diff
@@ -96,6 +96,15 @@ func GetServersEligible(w http.ResponseWriter, r *http.Request) {
 		api.WriteResp(w, r, v3ServerList)
 		return
 	}
+	if inf.Version.Major == 4 && inf.Version.Minor < 1 {
```

Comment:
This causes APIv5 to return APIv4.0 structs instead of 4.1 - should check if API version is >= 4.1 instead of >=4.1 and < 5.0

Reply:
Ah yeah, forgot that we have API v5 now. I'll get that fixed.

Reply:
I see the `4.1` structs being returned for apiv5 calls. This check is for version 4.0 calls. All others(4.1 and 5.0) will default to 4.1. Are you seeing something different?

Reply:
Oh, no my mistake, I simply read it wrong